Microsoft.Toolkit.Uwp.UI.Controls/RangeSelector/RangeSelector.cs

```diff
@@ -623,50 +623,18 @@ private static void RangeMaxChangedCallback(DependencyObject d, DependencyProper
 
         private static void UpdateToolTipText(RangeSelector rangeSelector, TextBlock toolTip, double newValue)
         {
-            if (rangeSelector.StepFrequency < 1)
-            {
-                toolTip.Text = string.Format("{0:0.00}", newValue);
-            }
-            else
-            {
-                toolTip.Text = string.Format("{0:0}", newValue);
-            }
-        }
-
-        private static void UpdateMinimumDisplayText(double newValue, RangeSelector rangeSelector)
-        {
-            // Safety check in case the template is missing the _minValueText element
-            if (rangeSelector._minValueText == null)
-            {
-                return;
-            }
-
-            if (rangeSelector.StepFrequency < 1)
-            {
-                rangeSelector._minValueText.Text = string.Format("{0:0.00}", newValue);
-            }
-            else
-            {
-                rangeSelector._minValueText.Text = string.Format("{0:0}", newValue);
-            }
+            toolTip.Text = string.Format("{0:0.##}", newValue);
         }
 
-        private static void UpdateMaximumDisplayText(double newValue, RangeSelector rangeSelector)
+        private static void UpdateDisplayText(TextBlock valueDisplayTextblock, double newValue)
```

Comment:
This causes a weird "issue" where the size of the element changes as the value changes and causes the control to look jittery. I think we need to keep the size somewhat constant and always show the decimal points if the StepFrequency is a fraction.